centreon/src/Core/Service/Domain/Model/SimpleEntity.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Service\Domain\Model;
+
+use Assert\AssertionFailedException;
+use Centreon\Domain\Common\Assertion\Assertion;
+use Core\Common\Domain\TrimmedString;
+
+class SimpleEntity
```

Comment:
"Simple Entity" of what ?
The naming is not convincing me. I need to think about a better idea.

Reply:
It could be anything: a template, a category ... it's basicaly just an id and a name
I took that idea from Laurent's PR on FindHosts